test/jdk/javax/swing/text/html/CSS/bug4174871.java

```diff
@@ -50,7 +50,6 @@ public static void main(String[] args) throws Exception {
 
             SwingUtilities.invokeAndWait(bug4174871::testUI);
             robot.waitForIdle();
```

Comment:
`waitForIdle` isn't necessary either: the value of `passed` has been set inside, it cannot change — nothing's left to wait for.

It applies to all the tests.